.github/workflows/build.yml

```diff
@@ -250,7 +250,7 @@ jobs:
     needs: [tests, config-tests]
     name: Publish branch build
     runs-on: ubuntu-22.04
-    if: ${{ github.event_name != 'pull_request' }}
+    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
This is unfortunate but needed so I can get the PR number, otherwise GH has no idea what I'm talking about. I _think_ it's still fine, it just means you don't publish unless you have a PR rather than publishing every branch.